src/overlays/actors/ovl_En_Time_Tag/z_en_time_tag.h

```diff
@@ -7,10 +7,57 @@ struct EnTimeTag;
 
 typedef void (*EnTimeTagActionFunc)(struct EnTimeTag*, PlayState*);
 
-#define ENTIMETAG_GET_1F(thisx) ((thisx)->params & 0x1F)
-#define ENTIMETAG_GET_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
-#define ENTIMETAG_GET_1FE0(thisx) (((thisx)->params & 0x1FE0) >> 0x5)
-#define ENTIMETAG_GET_E000(thisx) (((thisx)->params & 0xE000) >> 0xD)
+#define TIMETAG_GET_TYPE(thisx) (((thisx)->params & 0xE000) >> 0xD)
+
+#define TIMETAG_GET_KICKOUT_TEXT_TYPE(thisx) (((thisx)->params & 0x1FE0) >> 0x5)
+#define TIMETAG_GET_KICKOUT_EXIT_INDEX(thisx) ((thisx)->params & 0x1F)
+#define TIMETAG_KICKOUT_HOUR(thisx) ((thisx)->home.rot.x)
+#define TIMETAG_KICKOUT_MINUTE(thisx) ((thisx)->home.rot.y)
+
+#define TIMETAG_GET_SOARING_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
+
+#define TIMETAG_GET_DIARY_TYPE(thisx) ((thisx)->params & 0x7F)
+#define TIMETAG_GET_DIARY_TIMER(thisx) ((thisx)->home.rot.x)
+#define TIMETAG_GET_DIARY_SONG(thisx) ((thisx)->home.rot.z)
```

Comment:
```suggestion
#define TIMETAG_DIARY_TIMER(thisx) ((thisx)->home.rot.x)
#define TIMETAG_DIARY_SONG(thisx) ((thisx)->home.rot.z)
```
Looks like these also get set to as well.